spec/truffle/process/euid_spec.rb

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 2.0, or
+# GNU General Public License version 2, or
+# GNU Lesser General Public License version 2.1.
+
+require_relative '../../ruby/spec_helper'
+
+describe "Process.euid=" do
+
+  platform_is_not :windows do
+    as_superuser do
```

Comment:
`as_superuser` specs are almost never run, because it's very inconvenient (basically need to run them in Docker as root).
Could you try using a regular (non-superuser) spec and use `Etc.getlogin` to find the current user?
Then I think it should work even without superuser rights.

Reply:
I see, I will adjust the test case.

Reply:
I am running into a rather strange issue. If I change the test case to set the `euid` to equal to the process' real `uid` and run the suit as a non-root user, then I will get an error like this whenever I run the testing using the `native` and `jvm` build:
```
[/ |                   0%                     | 00:00:00]      0F      0E <internal:core> core/process.rb:407:in `const_missing': uninitialized constant Etc (NameError)`
```

I do not encounter the same issue with the adjusted test case when running as root (using `as_superuser`) or when running the test case as a regular user but using the MRI. Adding a `require 'etc'` line inside of `Process.euid=` seems to fix the issue, but I am not sure if that should be done.

I will push the code to the PR to make it easier to see the problem.

Reply:
https://github.com/oracle/truffleruby/pull/2651#discussion_r863612807

Reply:
Thank you, I did not notice the comment.